frontend/packages/ceph-storage-plugin/src/constants/queries.ts

```diff
@@ -54,13 +54,13 @@ export const CAPACITY_USAGE_QUERIES = {
   [StorageDashboardQuery.CEPH_CAPACITY_TOTAL]: 'ceph_cluster_total_bytes',
   [StorageDashboardQuery.CEPH_CAPACITY_USED]: 'ceph_cluster_total_used_bytes[60m:5m]',
```

Comment:
@aruniiird I think we don't need the timer over here. 
```suggestion
  [StorageDashboardQuery.CEPH_CAPACITY_USED]: 'ceph_cluster_total_used_bytes',
```
@anmolsachan please confirm.

Reply:
Yes, Even I thought so, There should not be need here for time.

Reply:
that query is also used for Utilization card and Capacity in Overview dashboard - see https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/plugin.ts#L164 .Thats why the query has time range specified

Reply:
@rawagner, isn't it a good idea to put the range (added to the query) where ever required and keep the original as the plain one. Something like this,

In CAPACITY_USAGE_QUERIES
```typescript
[StorageDashboardQuery.CEPH_CAPACITY_USED]: 'ceph_cluster_total_used_bytes'
```

and in the above said line, we use something like,
```typescript
query: `${CAPACITY_USAGE_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_USED]}[60m:5m]`,
```

Reply:
Im fine with adding range only when required

Reply:
Thanks @rawagner ... I will make the changes... (=

Reply:
@aruniiird I agree with your approach, We can put it where ever it's required. 
@rawagner trying to understand why is it required in overview dashboard?

Reply:
Storage query is different for RHHI - we want to show ceph size/usage instead of basic node filesystem

Reply:
@rawagner so are we using it for the capacity donut or somewhere else also?

Reply:
capacity and utilization card - area chart (thats why we need time range)

Reply:
Thanks, got it.